ui/src/schemas/index.ts

```diff
@@ -64,3 +64,9 @@ export const issueSeverity: z.ZodEnum<[Severity, ...Severity[]]> = z.enum([
 export const issueSeveritySchema = z.object({
   severity: z.array(issueSeverity).optional(),
 });
+
+// Enum schema for the possible values of the rule violation severities
+export const ruleViolationSeverity = issueSeverity;
+
+// Rule violation severity schema that is used for search parameter validation
+export const ruleViolationSeveritySchema = issueSeveritySchema;
```

Comment:
I would just use the same schema, no need for re-exports with different name.

Reply:
Wouldn't this make the code harder to read?

Reply:
@mmurto Sorry, but I'd leave this as-is, as the schema naming has been used in many other tables, and I'd like to create a follow-up PR, where the naming will be fixed for all. I can create the follow-up PR shortly after this one.